src/components/Inputs/DescInput/DescInput.test.tsx

```diff
@@ -0,0 +1,115 @@
+import { render, fireEvent, screen } from '@testing-library/react';
+import DescInput from './DescInput';
+import { matchers } from '@emotion/jest';
+
+expect.extend(matchers);
+
+describe('DescInput', () => {
+  const onChange = jest.fn();
+  const renderDescInput = () => {
+    const initialProps = { onChange };
+    return render(<DescInput {...initialProps} />);
+  };
+
+  describe('focus되면', () => {
+    it('인풋라인이 파란색으로 바뀐다', async () => {
+      const { getByRole } = renderDescInput();
+      const descInput = getByRole('textbox');
+      expect(descInput).toBeInTheDocument();
+
+      expect(descInput).toHaveStyleRule('border', '1px solid #3c3e44');
+
+      fireEvent.keyDown(descInput, { key: 'Enter' });
+
+      // await expect(descInput).toHaveStyleRule('border', '1px solid #3941ff');
+    });
+  });
+
+  describe('입력하면', () => {
+    it('onChange에 값이 들어간다', async () => {
+      const { getByRole } = renderDescInput();
+      const descInput = getByRole('textbox');
+      expect(descInput).toBeInTheDocument();
```

Comment:
```suggestion
      const descInput = getByRole('textbox');
```

이건 불필요한 검증같네요. 어짜피 없으면 밑에서 다 걸릴꺼에요

Reply:
중복된 테스트는 안하는게 좋아요. 테스트가 개발 생산성에 발목을 잡아버리거든요

Reply:
아하 중복될만한 테스트는 줄여보도록 하겠습니다!